meson.build

```diff
@@ -561,16 +561,19 @@ ggml_dep = dummy_dep
 if get_option('enable-ggml')
   message('preparing ggml')
 
+  if get_option('platform') != 'android' and get_option('platform') != 'tizen'
+    ggml_dir_relative = subprojects_dir_relative / 'ggml'
+    ggml_patch_relative = subprojects_dir_relative / 'packagefiles' / 'ggml' / '0001-nntrainer-ggml-patch.patch'
+    ggml_patch_command = ['git', 'apply', '--directory=' + ggml_dir_relative, ggml_patch_relative]
+    run_command(ggml_patch_command, check: false)
```

Comment:
Changing `check: true` will cause the build to stop if the patch fails. 
Is this what you intended?